tests/console/w3m.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests - FIPS tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Case 1525204 - FIPS: w3m
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run() {
+    my $name = ref(@_[0]);
+    select_console "root-console";
+
+    assert_script_run('rpm -q w3m');
+    save_screenshot;
+    script_run("zypper --no-refresh se -it pattern fips");
+
+    send_key "ctrl-l";
+    script_run "w3m https://www.google.com", 0;
+    assert_screen "w3m-connect-google-webpage";
+    send_key "q";
+    sleep 1;
+    send_key "y";
+
+    send_key "ctrl-l";
+    script_run "w3m https://www.suse.com", 0;
+    assert_screen "w3m-connect-suse-webpage";
+    send_key "q";
+    sleep 1;
+    send_key "y";
+
+    send_key "ctrl-l";
+    script_run "w3m https://build.opensuse.org", 0;
+    assert_screen "w3m-connect-OBS-webpage";
+    send_key "q";
+    sleep 1;
+    send_key "y";
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).